memgpt/agent.py

```diff
@@ -566,7 +567,7 @@ def handle_ai_response(self, response_message):
                 function_response = package_function_response(True, function_response_string)
                 function_failed = False
             except Exception as e:
-                error_msg = f"Error calling function {function_name} with args {function_args}: {str(e)}"
+                error_msg = f"Error calling function {function_name} with args {function_args}: {str(e)}\n{traceback.format_exc()}"

```

Comment:
Do we want to pass this much data back into the response to MemGPT? It might be excessive.

What about printing the traceback for the user, but only sending the `str(e)` for MemGPT?